### ISSUE TITLE
Can't match Discord channel "global"

### ISSUE THREAD

**Issue by momoservertw** [kangarko/ChatControl-Pro#1001]
Hi, I have a problem that player's message can send to discord, but discord can not.
If I turn the option "Deny_Unknown_Channel" to false, it would not cause this issue. But the plugin will not control Discord message any more.

ChatControlPro version: 7.10.5
DiscordSRV version: 1.16.4 
Spigot version: git-Spigot-dcd1643-e60fc34 (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT)

ChatControl\setting.yml
![image](https://user-images.githubusercontent.com/38014059/49329846-6a417200-f5c0-11e8-85d0-8f275be4e8b3.png)

DiscordSRV\config.yml
![image](https://user-images.githubusercontent.com/38014059/49329851-847b5000-f5c0-11e8-9287-d14414761802.png)

I almost use the default settings and it works fine before using ChatControlPro.

**Comment by kangarko** [comment]
Are you saying that a message from player goes to Discord, but from Discord does not go to the in-game chat?

If this is the case, please set Debug to ["discord"] and post me your console messages when you send a message on Discord.

**Comment by momoservertw** [comment]
Hey, after I used Debug mode, it shows that the channel does not match.
[20:17:18 INFO]: [ChatControl debug] Private messaging commands injected.
[20:17:30 INFO]: [ChatControl] [discord] Received Discord message from Discord's '茉茉的國度' channel: test
[20:17:30 INFO]: [ChatControl] [discord] > Message not being sent to admin chat (configuration requires admin chat name: ?????)
[20:17:30 INFO]: [ChatControl] [discord] > Message NOT SENT ANYWHERE because such channel is missing in ChatControl.

ChatControl and DiscordSRV have the same channel name in Config, but this problem occurs when Discord's channel is different. ![image](https://user-images.githubusercontent.com/38014059/49648212-e8d65d80-fa5f-11e8-998e-f4563fffa76c.png)

When I changed the name of the Discord channel, there was no problem. Is it possible to load two plugins' config instead of reading Discord's. So it will not happen in a similar situation.
